packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -132,19 +136,24 @@ class MessageGroup extends Widget<Properties> {
       .addClass(CHAT_MESSAGEGROUP_TIME_CLASS)
       .appendTo($information);
 
-    if (isDefined(timestamp)) {
-      $time.text(this._getTimeValue(timestamp));
+    const timeValue = this._getTimeValue(timestamp);
+    if (timeValue) {
+      $time.text(timeValue);
     }
 
     $information.appendTo(this.element());
   }
 
-  _getTimeValue(timestamp: Date | string | number): string {
-    const options: Intl.DateTimeFormatOptions = { hour: '2-digit', minute: '2-digit', hour12: false };
-    const date = dateSerialization.deserializeDate(timestamp);
+  _getTimeValue(timestamp: Date | string | number | undefined): string | boolean {
+    const deserializedDate = dateSerialization.deserializeDate(timestamp);
 
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-    return date.toLocaleTimeString(undefined, options);
+    if (!isDate(deserializedDate) || isNaN(deserializedDate.getTime())) {
+      return false;
+    }
+    const { messageTimestampFormat } = this.option();
+    const formattedTime = dateLocalization.format(deserializedDate, messageTimestampFormat);
+
+    return `${formattedTime}`;
```

Comment:
```suggestion
    return formattedTime;
```

Reply:
It strange to see that you use templated literals for type conversion here. Let's improve it and choose other way to avoid ts issues

Reply:
https://github.com/DevExpress/DevExtreme/pull/28234/commits/146df9b953cfca2760258090346ce0e5f1173871